dessia_common/workflow/core.py

```diff
@@ -118,14 +118,55 @@ def copy(self, deep: bool = False, memo=None):
         """ Copies and gives a new object with no linked data. """
         return TypedVariable(type_=self.type_, name=self.name)
 
+    def get_script_elements(self, script: ToScriptElement) -> ToScriptElement:
```

Comment:
I don't understand what is done here. Code is too complex to be merged. Consider having a look at the schema module that might have a lot of features you need here.